E-commerce/style.css

```diff
@@ -41,11 +41,17 @@
 /* Products */
 .best-selling img {
   padding: 10px;
-  height: 300px;
+  height: 100%;
   margin: 0 auto;
   display: block;
   padding-bottom: 50px;
+  width: 100%;
+}
 
+.img-container{
+  height: 50%;
+  width: 50%;
+  margin: 25px auto;
```

Comment:
Updating this to have a height and width of `50% `and a margin of `25px auto` will help with the layout of the images in the product boxes